moss_cli_demo.py

```diff
@@ -34,7 +34,7 @@
 
 def clear():
     os.system('cls' if platform.system() == 'Windows' else 'clear')
-    
+
```

Comment:
Trailing spaces were trimmed by my editor. I can revert this if requested in the code review.